skywalking/client/__init__.py

```diff
@@ -28,6 +28,9 @@ class TraceSegmentReportService(object):
     def report(self, generator):
         raise NotImplementedError()
 
+class MeterReportService(object):
+    def report(self, generator):
```

Comment:
also report_batch?

Reply:
@jiang1997 I collected the call times of an interface through Counter and reported it to skywalking. Now I want to display this metric on the ui page. How do I do it? Thank you!

Reply:
Learn MAL and Dashboard setup.
- https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-meter/
- https://skywalking.apache.org/docs/main/next/en/concepts-and-designs/mal/
- Examples, https://github.com/apache/skywalking/tree/89d14e6256338840cd4a9b37f76980c8558e50c3/oap-server/server-starter/src/main/resources/meter-analyzer-config/
- Dashboard, https://skywalking.apache.org/docs/main/next/en/ui/readme/

Reply:
@March225
This is just a simple supplement to wu-sheng's advice for the sake of simplicity.
1. Try to append your metric's definition to the server-side [config](https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/meter-analyzer-config/python-runtime.yaml#L31-L36) file like below
https://github.com/apache/skywalking/blob/e2f1c0eadb52082e16f3abfc29dd8860000b596b/oap-server/server-starter/src/main/resources/meter-analyzer-config/python-runtime.yaml#L31-L36
Then edit the dashboard and add new widget to show your metric.
2. If nothing goes wrong, you should find your metric here.
![image](https://user-images.githubusercontent.com/28537230/222797852-9a942c0e-2a5f-4e47-88de-0650c48dd2be.png)